src/accumulator.jl

```diff
@@ -12,6 +12,16 @@ counter(T::Type) = Accumulator(T,Int)
 Accumulator{T,V<:Number}(dct::Dict{T,V}) = Accumulator{T,V}(copy(dct))
 counter{T}(dct::Dict{T,Int}) = Accumulator{T,Int}(copy(dct))
 
+"""
+```
```

Comment:
I haven't looked at the rest, but the ` are not needed. OTOH, add 4 spaces before code lines (such as signatures).